aws-frauddetector-detector/src/aws_frauddetector_detector/helpers/update_worker_helpers.py

```diff
@@ -35,6 +35,7 @@ def validate_dependencies_for_detector_update(
         if not get_event_types_succeeded:
             raise exceptions.NotFound("detector.EventType", event_type_name)
     validation_helpers.validate_external_models_for_detector_model(afd_client, model)
+    validation_helpers.validate_model_versions_for_detector_create(afd_client, model)
```

Comment:
nit: just realized we did not rename this method. Since we use this for `detector_update` as well as `detector_create`, should we rename to `validate_model_versions_for_detector_model` ?

Reply:
Agreed. Revising